src/hydra-queue-runner/state.hh

```diff
@@ -489,7 +490,7 @@ private:
         bool & stepFinished);
 
     Jobset::ptr createJobset(pqxx::work & txn,
-        const std::string & projectName, const std::string & jobsetName);
+        const std::string & projectName, const std::string & jobsetName, const int & jobsetID);
```

Comment:
`const int &` isn't very common, usually that'd be an `int` or `const int`

Reply:
Thanks! Do you have a recommendation re const int vs int?

Reply:
we'd use `const int` because immutability by default is nice.